vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -195,12 +196,19 @@ public VimeoAccount getVimeoAccount() {
 
     public void setVimeoAccount(@Nullable VimeoAccount vimeoAccount) {
         if (vimeoAccount == null) {
-            vimeoAccount = new VimeoAccount();
+            vimeoAccount = new VimeoAccount(this.configuration.accessToken);
+            this.configuration.saveAccount(vimeoAccount, null, null);
```

Comment:
So before it was just an empty constructor that did nothing (except create an object). So instead I modified the constructor so it can take a nullable access token, so that if it's null, it'll be effectively the same as an empty constructor.

We only needed the empty constructor for this case, so I figured I could remove the need for one entirely with this functionality.